adapters/coinzilla/coinzilla.go

```diff
@@ -0,0 +1,101 @@
+package coinzilla
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+)
+
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters.Bidder, error) {
+	bidder := &coinzillaAdapter{
+		endpoint: config.Endpoint,
+	}
+	return bidder, nil
+}
+
+type coinzillaAdapter struct {
+	endpoint string
+}
+
+func (adapter *coinzillaAdapter) MakeRequests(openRTBRequest *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) (requestsToBidder []*adapters.RequestData, errs []error) {
+	if len(openRTBRequest.Imp) == 0 {
+		return nil, []error{&errortypes.BadInput{
+			Message: "No impression in the bid request",
+		}}
+	}
+	if len(openRTBRequest.Imp) > 0 {
+		var imp = &openRTBRequest.Imp[0]
+		var bidderExt adapters.ExtImpBidder
+		if err := json.Unmarshal(imp.Ext, &bidderExt); err != nil {
+			return nil, []error{&errortypes.BadInput{
+				Message: err.Error(),
+			}}
+		}
+		var ttxExt openrtb_ext.ExtAdpone
```

Comment:
Why use `openrtb_ext.ExtAdpone` here rather than `ExtImpCoinzilla` you created as part of this PR?

Reply:
Updated.